notebooks/notebooks/iclr2022/baseline-attacks.ipynb

```diff
@@ -41,30 +41,32 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #62.                    "cp127": "\\texttt{cp127}",
</pre><p>Might be easier to just write <code>r"\texttt{cp127}</code> (and similarly for the following lines) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>